.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.env
```

Comment:
please remove this we want to know if a `.env` file gets pushed or someone made a pr that relies on the `.env` file, because we should not have one ever in this repo.